tests/mocks/account_retriever.go

```diff
@@ -0,0 +1,39 @@
+package mocks
+
+import (
+	gomock "github.com/golang/mock/gomock"
+	reflect "reflect"
+)
+
+type MockNodeQuerier struct {
```

Comment:
missing comments on this file

Reply:
This file is autogenerated by `mockgen`

Reply:
Is it autogenerated once only? I don't see the problem of adding comments on top of it

Reply:
In all honesty, I can't see any benefit in adding a comment line either, plus it would be overwritten in case we need to regenerate the file. This piece of code doesn't target clients, docs is provided by gomock's godoc. What I can do is adding a package comment as there's a flag in `mockgen` for that.